src/lib.rs

```diff
@@ -261,6 +261,30 @@ impl FixedBitSet
             other: other,
         }
     }
+
+    /// In-place union of two `FixedBitSet`s.
+    pub fn union_with(&mut self, other: &FixedBitSet)
+    {
+        if other.len() >= self.len() {
+            self.grow(other.len());
+        }
+        let mn = std::cmp::min(self.data.len(), other.data.len());
+        for i in 0..mn {
+            self.data[i] |= other.data[i];
+        }
```

Comment:
Looks good. I'm not sure these loops codegen fine like this (do you know)? I'd prefer .zip() for idiomaticity and performance.